src/LuaTransformer.ts

```diff
@@ -3537,9 +3538,9 @@ export class LuaTransformer {
         throw TSTLErrors.CouldNotFindEnumMember(enumDeclaration, memberName, tsOriginal);
     }
 
-    public transformStringCallExpression(node: ts.CallExpression): tstl.Expression {
+    public transformStringCallExpression(node: ts.CallExpression, signature: ts.Signature): tstl.Expression {
```

Comment:
Thinking about the API we want to make available for people to override the transformers, I have my doubts if adding this signature parameter is a good idea. We have a couple options here: Can we just get the signature from the node inside this function? Is this a function that should not be in the public interface? Or do we just keep it like this.

My main point being it is not really clear what this signature is or where you should get it from? Is it not just part of the call expression?

Reply:
That's a good point. I'm pretty sure we can grab it inside the function - I was only passing it in since we already had it.